front/src/pods/master/master.component.tsx

```diff
@@ -131,6 +131,13 @@ const VotingInProgressComponent: React.FC<VotingInProgressProps> = props => {
       <div className={cx(classes.containerComponent, classes.leftContainer3)}>
         <TablePlayerComponent playersCollection={playerVotingStatus} />
       </div>
+      <div className={cx(classes.lector)}>
```

Comment:
The idea about this component is to create a common component with similar behaviour to Snackbar but it will available only for screen readers:

![snackbar](https://user-images.githubusercontent.com/4374977/111438364-70fa4180-8704-11eb-8586-f021807e249e.gif)

For example, you can create `ScreenReaderSnackbar` in common components to show any message during any time (pseudocode):

```
interface Props {
  // We will have many messages to show, for example, many users voting at the same time
 // Each message will be available only for 1 second (we can provide the time in seconds by props)
  messages: string[];
}


{messages.map(message => (
          <div aria-live="assertive" aria-atomic="true">
            <p>{message}</p>
          </div>
        ))}
```

- We can create a custom hook similar to `snackbar.hook` to show a message when we receive something from socket. Example of use:

_./pods/master/master.container.tsx_

```diff
....

+  const { showMessage } = useScreenReaderSnackbar();

....

socket.on(SocketOuputMessageLiteral.MESSAGE, msg => {
      if (msg.type) {
        const { type, payload } = msg;

        switch (type) {
          case SocketInputMessageTypes.CONNECTION_ESTABLISHED_PLAYER:
            const newPlayerCollection = AddNewPlayer(
              playerCollectionRef.current,
              payload
            );
+          showMessage(`El usuario ${payload} se ha conectado`);

            updatePlayerCollection(newPlayerCollection);
            break;
          case SocketInputMessageTypes.NOTIFY_USER_VOTED:
            const updatedPlayerList = userVoted(
              playerCollectionRef.current,
              payload
            );
+          showMessage(`El usuario ${payload} ha votado`);
            updatePlayerCollection(updatedPlayerList);
            break;
          case SocketInputMessageTypes.SHOW_VOTING_RESULTS:
            // refactor this to a map
            const playerVoteResults = msg.payload.map(voteResult => ({
              ...voteResult,
              voted: !!voteResult.vote,
            }));
            // ***
            setPlayerCollectionVoteResult(playerVoteResults);
            setMasterStatus(MasterStatus.SHOWING_RESULTS);
            break;
          case SocketInputMessageTypes.USER_DISCONNECTED:
            //TODO Refresh list of users in master room
            updatePlayerCollection(
              playerCollectionRef.current.filter(p => p.nickname !== payload)
            );
+          showMessage(`El usuario ${payload} se ha desconectado`);
            break;
        }
      }
    });

```

- As the same way, we can use in `player.container`

- Check `./src/common/components/snackbar` implementation, maybe we can use similar approach (context + hook + component).